crypto/dh/dh_key.c

```diff
@@ -230,3 +230,39 @@ static int dh_finish(DH *dh)
     BN_MONT_CTX_free(dh->method_mont_p);
     return 1;
 }
+
+int DH_buf2key(DH *dh, const unsigned char *buf, size_t len)
```

Comment:
Since this is now internal, please can we call it `dh_buf2key` instead? We prefer to keep the capitalised form for public APIs.